packages/http/src/validator/__tests__/functional.spec.ts

```diff
@@ -18,6 +22,12 @@ describe('HttpValidator', () => {
       it('returns validation errors for whole request structure', async () => {
         expect(await validator.validateInput({ resource: httpOperations[2], input: BAD_INPUT })).toMatchSnapshot();
       });
+
+      describe('when all required params are provided', () => {
+        it('returns no validation errors', async () => {
+          expect(await validator.validateInput({ resource: httpOperations[0], input: GOOD_INPUT })).toEqual([]);
```

Comment:
```suggestion
          expect(validator.validateInput({ resource: httpOperations[0], input: GOOD_INPUT })).resolves.toEqual([]);
```

Reply:
@XVincentX, If only `husky` allowed me to do this ;>

Reply:
I'm not following, what has husky to do with this?

Reply:
```
ERROR: /home/lag-of-death/RubymineProjects/prism/packages/http/src/validator/__tests__/functional.spec.ts:28:11 - Promises must be handled appropriately
ERROR: /home/lag-of-death/RubymineProjects/prism/packages/http/src/validator/__tests__/functional.spec.ts:58:11 - Promises must be handled appropriately
husky > pre-commit hook failed (add --no-verify to bypass)
```

Reply:
It's not husky, that's ESLint and it's actually right. I've modified the suggestion.

Reply:
+ you can remove `async` since there is no `await` inside

Reply:
@XVincentX, but now I have to know to return that `promise`. With `async/await` it is much more clear.

Reply:
I'm gonna go with `async/await` unless you think it's crucial to change it. And btw, all of the other tests are `async/await`. So let's stay consistent here as the alternatives are not in any way better.

Reply:
It's not crucial, it can stay as it is!